### PR TITLE
Adding the Praktomat Public Test

### LINKED ISSUE #108: Invalid Argument Format in #106
The rule format in invalidArgumentFormatTest and duplicateArgumentsTest is invalid.
It should be structured as "rule=45-45-45-45-45" not "rule={45,45,45,45,45}". [source:Praktomat public test].

### PR DESCRIPTION
**Pull request by MartinLoeper**
fixes #108

### CODE REVIEW

File: .classpath [hunk 1]
```diff
@@ -1,7 +1,11 @@
...
-	<classpathentry kind="con" path="org.eclipse.jdt.junit.JUNIT_CONTAINER/4"/>
-	<classpathentry kind="output" path="bin"/>
-</classpath>
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" path="src"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8">
+		<attributes>
+			<attribute name="owner.project.facets" value="java"/>
+		</attributes>
```
Comment (pcworld) [inline .classpath:7]: Is this change intended?

Comment (MartinLoeper) [inline .classpath:7]: no, somethng went wrong during the push. I have a look at it. ;)
